google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfiguration.java

```diff
@@ -130,6 +139,78 @@ public boolean isStagedArtifactNameLegacy() {
     return stagedArtifactNameLegacy;
   }
 
+  @Attribute("compileEncoding")
+  public String getCompileEncoding() {
```

Comment:
There are many staging properties, do we know if it's possible to introduce a helper bean, like `StagingProperties`, and annotate it with `@Attribute`? It could possibly make it a bit simpler to understand major properties in the deployment configuration then.

Reply:
this sounds like a good suggestion. maybe add a todo to revisit this in a future PR?